JSDemos/Demos/DataGrid/GroupSummaries/description.md

```diff
@@ -1 +1,17 @@
\ No newline at end of file
+To configure group summaries, populate the [summary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/).[groupItems](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/) array with summary configuration objects. Each object should specify a [column](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#column) that supplies data for summary calculation and a [summaryType](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#summaryType)&mdash;an aggregate function that should be applied to this data.
+
+### Position a Group Summary
+
+Group summaries are displayed in parentheses after the group caption. You can reposition them as follows:
+
+* Display a summary in a column          
+You can align a summary with the column by which it was calculated (see the Sale Amount and Total Amount columns). To do this, enable the [alignByColumn](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#alignByColumn) property. If you want to align a summary with a column of your choice, specify it in the [showInColumn](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#showInColumn) property.
+
+* Move a summary to the group footer       
+To position a summary in the group footer, enable the [showInGroupFooter](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#showInGroupFooter) property. The summary will be aligned with the column by which it was calculated (see the Total Amount column). If you want to use another column, specify it in the [showInColumn](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#showInColumn) property.
+
+### Format Summary Values
+You can format a summary value in the [valueFormat](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#valueFormat) property. This demo specifies the *"currency"* **valueFormat** for the Total Amount and Sale Amount summary values. You can also use the [displayFormat](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#displayFormat) property to set a custom display format for summary values. For example, you can add custom text to a summary value, as shown in this demo for summaries of the *"sum"* and *"count"* types.
```

Comment:
Format something "IN" the property - this preposition doesn't work. 

You also say "a summary value", but we need to refer to multiple values, since we're talking about group summaries. 

`Total Amount and Sale Amount summary values` - this phrase is a bit misleading. Those a column names, so it's the format of summary values calculated against those columns. 

You don't explain what is "a custom display format". You give one example, which is "add text". But what else? If it's only that (in addition to actually formatting the value), then I give a suggestion below. 

```suggestion
To format summary values, use the [valueFormat](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#valueFormat) property. This demo applies the *"currency"* format to summary values calculated against the Total Amount and Sale Amount columns. If you also want to add custom text to summary values, use the [displayFormat](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/groupItems/#displayFormat) property. This demo sets this property for summaries of *"sum"* and *"count"* types.
```